main/bubblewrap/APKBUILD

```diff
@@ -34,6 +35,11 @@ build() {
 	make
 }
 
+check() {
+	cd "$builddir"
+	make check
```

Comment:
Seems this is not working as expected:
```
SKIP: tests/test-run.sh - Seems like bwrap is not working at all. Maybe setuid is not working
============================================================================
Testsuite summary for bubblewrap 0.3.3
============================================================================
# TOTAL: 1
# PASS:  0
# SKIP:  1
# XFAIL: 0
# FAIL:  0
# XPASS: 0
# ERROR: 0
============================================================================
```

It probably should be run as regular user (not fakeroot or root).